bindings/c/test/mako/mako.cpp

```diff
@@ -136,28 +136,31 @@ int cleanup(Database db, Arguments const& args) {
 	} else {
 		int batch_size = args.tenant_batch_size;
 		int batches = (args.total_tenants + batch_size - 1) / batch_size;
+		fdb::TypedFuture<fdb::future_var::ValueRef> tenantResults[batch_size];
+		// Issue all tenant reads first
+		Transaction getTx = db.createTransaction();
+		for (int i = 0; i < args.total_tenants; ++i) {
+			std::string tenant_name = "tenant" + std::to_string(i);
+			tenantResults[i] = Tenant::getTenant(getTx, toBytesRef(tenant_name));
+		}
```

Comment:
oops, the initial declaration is supposed to be `tenantResults[args.total_tenants];`

Reply:
If you do it that way, then it will read all of the tenants in one transaction, which could be too many. I think you'll need to batch this part up as well.